website/content/configuration/_advanced_bgp_configuration.md

```diff
@@ -208,6 +208,52 @@ spec:
 In this way, all the IPs coming from `first-pool` will be reacheable only via `NodeA`
 and `NodeB`.
 
+### Announcing the Service to a subset of peers
+
+By default, every service IP is advertised to all the connected peers. It is possible
+to limit the set of peers a service IP is advertised to. This is achieved by using
+the `peers` field in the `BGPAdvertisement` CR.
+
+{{<mermaid align="center">}}
+graph TD
```

Comment:
This is not rendering fine in the docs (although it is in the mermaid tool):

![image](https://user-images.githubusercontent.com/200560/171008779-a0324396-27bb-459e-b15d-55339eea0aa4.png)

To test it, you need to install hugo and run `hugo server` locally.